release.sh

```diff
@@ -83,6 +83,18 @@ push_to_wordpress_svn() {
   svn commit -m "$SVN_COMMIT_MESSAGE"
 }
 
+  # Keep only the latest 20 tags
```

Comment:
I think we could even reduce this to 2 or 3.

Reply:
I wanted to be conservative :). Let's go with ten tags for now. We can lower it later if we need to.